source/_components/remote.harmony.markdown

```diff
@@ -77,6 +77,26 @@ Start an activity. Will start the default `activity` from configuration.yaml if
 | `entity_id`            |      yes | Only act on a specific remote, else target all.
 | `activity`             |      yes | Activity ID or Activity Name to start.
 
+##### {% linkable_title Example %}
+
+If in the 'harmony_REMOTENAME.conf' file you see for example:
+
+```bash
```

Comment:
Please use correct language reference.
In this case, it is not a language, so use `text` instead of `bash`.